src/openrct2/interface/Window_internal.h

```diff
@@ -48,7 +48,7 @@ struct rct_window
     rct_windownumber number{};
     uint16_t flags{};
     rct_scroll scrolls[3];
-    uint8_t list_item_positions[1024]{};
+    uint32_t list_item_positions[1024]{};
```

Comment:
have you checked all users of this field to make sure they werent abusing its size?

Reply:
The only actual use is the guest window, it has an assignment in multiplayer but its never used anywhere else really.